simple_test.py

```
@@ -0,0 +1,101 @@
+"""
+Simple test script to demonstrate Mad Spark Alt with custom prompts.
+"""
+
+import asyncio
+from typing import Optional, Any
+
+from mad_spark_alt.core import SmartQADIOrchestrator
+from test_utils import truncate_text, validate_qadi_result
+
+
+async def simple_qadi_test(problem: str, context: str = "") -> Optional[Any]:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `simple_qadi_test`

(ANN401)